staging_service/app.py

```diff
@@ -108,18 +104,31 @@ def _file_type_resolver(path: PathPy) -> FileTypeResolution:
         return FileTypeResolution(unsupported_type=ext)
 
 
+def _make_dts_file_resolver() -> Callable[[Path], FileTypeResolution]:
+    """Makes a DTS file resolver."""
+
+    def dts_file_resolver(path: PathPy):
+        # must be a ".json" file
+        file_parts = str(path).split(".")
+        ext = file_parts[-1]
+        if len(file_parts) < 2 or ext.lower() != "json":
+            return FileTypeResolution(unsupported_type=ext)
```

Comment:
```suggestion
        if path.suffix.lower() != "json":
            return FileTypeResolution(unsupported_type=ext)
```

Reply:
Leaving it as-is prevents a plain file called "json" from being counted as ok.

Reply:
Also, headslap, right, this is a pathlib Path so it has suffix built in.

Reply:
> Leaving it as-is prevents a plain file called "json" (without an extension) from being counted as ok.

This won't count `json` as ok because it doesn't have a suffix.

Although now I notice in that case it'll probably get a Nonetype error on the `.lower()` call

Reply:
Importing as PathPy threw me a little.
Path.suffix handles all that, though. It just returns an empty string, with is fine with `.lower()`

Reply:
yeah, I probably wouldn't have made a custom class called the same thing as a built in class myself

Reply:
Importing the JSON mapping item might be misleading. There's no guarantee it maps to the expected suffix string. ZIP certainly doesn't.

I'll do it, because I don't like magic strings either, but I think it's potentially misleading.

Reply:
In that case make a JSON constant in the current file and add a comment. Misleading code is worse than duplicated constants

Reply:
Ok, done.